### PR TITLE
Install non-dev SDL packages by default

### LINKED ISSUE #2599: Ubuntu packages should be packed with non-dev SDL scripts
We ship Ubuntu packages with installation of development version of SDL (`script/linux/install_sdl_1.sh` and `script/linux/install_sdl_2.sh` files). We should provide scripts for **normal** version of SDL instead.

### PR DESCRIPTION
**Pull request by qarkai**
Add separate scripts to install development packages.
Also don't remove other version of SDL before installation.
Fixes #2599.

### CODE REVIEW

File: script/linux/install_sdl_1.sh [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
```
Comment (ihhub) [inline script/linux/install_sdl_1.sh:1]: I would really recommend to just create 2 variants of the same script without an extra flag. Development should be as easy as possible while having a flag will definitely lead to cases when someone just forgets about it.

**Review by ihhub — COMMENTED** [review]
Hi @qarkai , I left one comment. Could you please reply to it?

File: script/linux/install_sdl_1.sh (hunk 1 above)
Comment (qarkai) [reply script/linux/install_sdl_1.sh:1]: Are `script/linux/install_sdl_1_dev.sh` and  `script/linux/install_sdl_2_dev.sh` ok?

Comment (oleg-derevenetz) [reply script/linux/install_sdl_1.sh:1]: I think yes.

Comment (ihhub) [reply script/linux/install_sdl_1.sh:1]: Yes. It's totally fine.

File: script/linux/install_sdl_1.sh [hunk 2]
```diff
@@ -1,10 +1,4 @@
...
 
-# Uninstall SDL 2
-sudo apt-get remove libsdl2-image-dev
-sudo apt-get remove libsdl2-mixer-dev
-sudo apt-get remove libsdl2-ttf-dev
-sudo apt-get remove libsdl2-dev
-
-# Install SDL 1.2 and other dependencies
-sudo apt-get install -y libsdl1.2-dev libsdl-ttf2.0-dev libsdl-mixer1.2-dev libsdl-image1.2-dev gettext
+# Install SDL 1.2
```
Comment (oleg-derevenetz) [inline script/linux/install_sdl_1.sh:3]: What if SDL2 libraries are already installed? Are SDL1 and SDL2 packages compatible on Ubuntu (can be installed simultaneously)? The same question regarding sdl and sdl-dev with the same version - can they be installed simultaneously? I suppose that answer to both questions is yes, but just to make sure (I don't have an Ubuntu close at hand to verify).

**Review by oleg-derevenetz — COMMENTED** [review]
Hi @qarkai I just left one question, please take a look when it will be convenient for you.

File: script/linux/install_sdl_1.sh (hunk 2 above)
Comment (qarkai) [reply script/linux/install_sdl_1.sh:3]: SDL1{-dev} and SDL2{-dev} don't conflict on Ubuntu. I mentioned that in e87a54a.

**Review by oleg-derevenetz — APPROVED** [review]

**Comment by ihhub** [comment]
Hi @qarkai and @oleg-derevenetz , I remember there was a reason why we had to remove SDL 1 if we install SDL 2. I'll check it soon.

**Comment by ihhub** [comment]
@qarkai , a bunch of thanks for improving the project!

**Pull request merged**